src/main/scala/org/globalforestwatch/summarystats/carbon_sensitivity/CarbonSensitivitySummary.scala

```diff
@@ -32,8 +34,17 @@ object CarbonSensitivitySummary {
                     row: Int,
                     acc: CarbonSensitivitySummary
                   ): CarbonSensitivitySummary = {
+
+        // Changes the lossYear type to PRODES if the sensitivity analysis is "legal_Amazon_loss"
+        val changeLossSource: String =
+          getAnyMapValue[String](acc.kwargs, "sensitivityType")
+
         // This is a pixel by pixel operation
-        val lossYear: Integer = raster.tile.loss.getData(col, row)
+        val lossYear: Integer = {
+          if (changeLossSource == "legal_Amazon_loss") raster.tile.lossLegalAmazon.getData(col, row)
```

Comment:
Codacy found an issue: [Consider using case matching instead of else if blocks](https://app.codacy.com/manual/thomas-maschler/gfw_forest_loss_geotrellis/pullRequest?prid=6075981)